packages/ag-charts-community/src/chart/data/dataModel.ts

```diff
@@ -1240,19 +1249,23 @@ export class DataModel<
                             `[${value}]`
                         );
                     }
-                    return INVALID_VALUE;
+                    reusableResult.value = INVALID_VALUE;
+                    return reusableResult;
                 }
+            } else {
+                reusableResult.valid = true;
             }
 
             if (def.processor) {
                 if (!processorFns.has(def)) {
                     processorFns.set(def, def.processor());
                 }
-                value = processorFns.get(def)?.(value);
+                value = processorFns.get(def)?.(value, idx);
```

Comment:
Small point - but it's faster to do `let value = processorFns.get(...)` then `if (value == null) { value = ...; processorFns.set(...) }` rather than `.has` then `.get`

Reply:
Fair point, done